include/pybind11/detail/internals.h

```diff
@@ -348,13 +494,21 @@ struct type_info {
     bool module_local : 1;
 };
 
-#define PYBIND11_INTERNALS_ID                                                                     \
-    "__pybind11_internals_v" PYBIND11_TOSTRING(PYBIND11_INTERNALS_VERSION)                        \
-        PYBIND11_COMPILER_TYPE_LEADING_UNDERSCORE PYBIND11_PLATFORM_ABI_ID "__"
+/// Information stored in a capsule on py::native_enum() types.
+struct native_enum_info {
+    const std::type_info *cpptype;
+    uint32_t size_bytes;
+    bool is_signed;
+
+    static const char *attribute_name() { return "__pybind11_enum__"; }
```

Comment:
How about `"__pybind11_native_enum__"`? (To not mix up terminology for `py::enum_` and `py::native_enum`.)